packages/core/admin/ee/server/controllers/admin.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const { env } = require('@strapi/utils');
+const { getService } = require('../../../server/utils');
+
+module.exports = {
+  async licenseLimitInformation() {
+    const permittedSeats = strapi.ee.licenseInfo.seats;
```

Comment:
where is this licenseInfo loaded? If 'seats' attribute does not exist, should we throw an error?

Reply:
It's loaded in Strapi class. If seats are not available, the controller will just return and seats will be undefined. This will stop the execution of license limit workflow. Don't really need to throw here as it is handled on line 9.